snapshot/git/gitdb/bundlestore.go

```diff
@@ -273,3 +273,22 @@ func (s *bundlestoreSnapshot) downloadBundle(db *DB) (filename string, err error
 func makeBundleName(key string) string {
 	return fmt.Sprintf("bs-%s.bundle", key)
 }
+
+func (b *bundlestoreBackend) storeFile(filePath string, ttl *bundlestore.TTLValue) (string, error) {
```

Comment:
Can you make uploadLocalSnapshots call this? Also I think you can just name this uploadFile to keep naming consistent. Same for db.go below.